charts/templates/statefulset.yaml

```diff
@@ -62,6 +108,34 @@ spec:
               {{- toYaml .Values.extraContainerPorts | nindent 12 }}
             {{- end }}
           lifecycle:
+            {{- if eq .Values.bootstrapHostName "" }}
+            postStart:
+              exec:
+                command:
+                  - bash
+                  - '-c'
+                  - |
+                    pid=$(ps aux | grep -i '/bin/bas[h] /usr' | awk {'print $2'})
+
+                    log () {
+                      local TIMESTAMP=$(date +"%Y-%m-%d %T.%3N")
+                      echo "${TIMESTAMP}  $@" > /proc/$pid/fd/1
+                    }
```

Comment:
need to add a logic here, if the MARKLOGIC_GROUP is "Default" or this node is not bootstrap node(the pod name is not ending with -0), then exit.

Reply:
added logic to check if group is Default and node is not bootstrap